src/common/mem.c

```diff
@@ -252,20 +323,29 @@ umempobj_get_rootptr(struct umem_pool *ph_p, size_t size)
 int
 umempobj_get_heapusage(struct umem_pool *ph_p, daos_size_t *curr_allocated)
 {
-	int rc;
+	PMEMobjpool		*pop;
+	struct dav_heap_stats	 st;
+	int			 rc = 0;
 
-	if (!use_bmem) {
-		PMEMobjpool *pop = (PMEMobjpool *)ph_p->up_priv;
+	switch (daos_md_backend) {
+	case DAOS_MD_PMEM:
+		pop = (PMEMobjpool *)ph_p->up_priv;
 
 		rc = pmemobj_ctl_get(pop, "stats.heap.curr_allocated",
 			     curr_allocated);
-	} else {
-		struct dav_heap_stats st;
-
+		break;
+	case DAOS_MD_BMEM:
 		rc = dav_get_heap_stats((dav_obj_t *)ph_p->up_priv, &st);
 		if (rc == 0)
 			*curr_allocated = st.curr_allocated;
+	case DAOS_MD_ADMEM:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment